src/styles/blocks/util/mixins.scss

```diff
@@ -0,0 +1,34 @@
+@mixin phone {
+  @media (min-width: 488px) {
+    @content;
+  }
+}
+
+@mixin tablet {
+  @media (min-width: 768px) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media (min-width: 1024px) {
+    @content;
+  }
+}
+
+@mixin font($size, $weight, $color, $line-height: false) {
+  font-family: $roboto;
```

Comment:
`font-family` can be specified once for the entire `body` tag, but now it is specified separately for each `class`